src/annotator/util/frame-util.js

```diff
@@ -15,29 +15,43 @@ export function findFrames(container) {
   return Array.from(container.querySelectorAll('iframe[enable-annotation]'));
 }
 
-// Check if the iframe has already been injected
+/**
+ * Check if the iframe has already been injected
+ *
+ * * @param {HTMLIFrameElement} iframe
+ */
 export function hasHypothesis(iframe) {
   return '__hypothesis' in iframe.contentWindow;
 }
 
-// Inject embed.js into the iframe
-export function injectHypothesis(iframe, scriptUrl, config) {
+/**
+ * Inject embed.js into the iframe
```

Comment:
```suggestion
 * Inject the client's boot script into the iframe
```

It may not be obvious here what embed.js refers to, since h's `/embed.js` route has a rather generic name.